sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/ConfigurationService.java

```diff
@@ -132,4 +217,18 @@ Mono<PagedResponse<ConfigurationSetting>> listKeyValueRevisions(
         @HeaderParam("Accept-Datetime") String acceptDatetime,
         @HeaderParam("Range") String range,
         Context context);
+
+    @Get("revisions")
+    @ExpectedResponses({200, 206})
+    @UnexpectedResponseExceptionType(HttpResponseException.class)
+    @ReturnValueWireType(ConfigurationSettingPage.class)
+    Mono<PagedResponse<ConfigurationSetting>> listKeyValueRevisionsSync(
```

Comment:
Is this meant to be sync or async? Name says sync but return is async

Reply:
Meant to delete this file. Deleted now.